apps/authentication/backends/saml2/views.py

```diff
@@ -271,7 +271,10 @@ def post(self, request):
             auth.login(self.request, user)
 
         logger.debug(log_prompt.format('Redirect'))
-        next_url = saml_instance.redirect_to(post_data.get('RelayState', '/'))
+        redir = post_data.get('RelayState', '/')
```

Comment:
redir = post_data.get('RelayState')

Default values are not necessary and can be misleading